tests/api-diff/run.t

```diff
@@ -28,3 +28,54 @@ there should be no diff:
 
   $ api-diff ref.cmi ref.cmi
   API unchanged!
+
+## Different .cmi files for type test:
+
+Here we generate a basic `.mli` files with two types and a function:
+
+### A file with an additional type:
+
+  $ cat > add_type.mli << EOF
+  > type t = int
+  > type added_t = float
+  > val f : t -> string
+  > EOF 
```

Comment:
Also, please don't leave any whitespaces. Here, whitespace after EOF is contributing to syntax error which hinders the compilation. Hence you are receiving this error:
```
+  File "add_type.mli", line 5, characters 0-0:
+  Error: Syntax error
+  [2]

Now we run api-watcher on the two cmi files, there should be a difference:

   $ api-diff ref.cmi add_type.cmi
-  API changed!
+  api-watcher: CMI argument: no 'add_type.cmi' file or directory
+  Usage: api-watcher [OPTION]… CMI CMI
+  Try 'api-watcher --help' for more information.
+  [124]
```
These cram tests are very specific about the syntax :p